src/Helper/Names/DefaultNamesHelper.php

```diff
@@ -91,4 +94,14 @@ private function byExpectedClassName(string $actualClass, string $expectedClass,
 
         return $this->byManifestClass($actualClass);
     }
+
+    private function makeManifest(string $manifestClass): ManifestInterface
```

Comment:
This totally will not work, since this requires for manifest to not have constructor with parameters.